include/mitsuba/render/phase.h

```diff
@@ -198,11 +271,11 @@ NAMESPACE_END(mitsuba)
 // -----------------------------------------------------------------------
 
 ENOKI_CALL_SUPPORT_TEMPLATE_BEGIN(mitsuba::PhaseFunction)
-    ENOKI_CALL_SUPPORT_METHOD(sample)
-    ENOKI_CALL_SUPPORT_METHOD(eval)
-    ENOKI_CALL_SUPPORT_METHOD(projected_area)
-    ENOKI_CALL_SUPPORT_METHOD(max_projected_area)
-    ENOKI_CALL_SUPPORT_GETTER(flags, m_flags)
+ENOKI_CALL_SUPPORT_METHOD(sample)
+ENOKI_CALL_SUPPORT_METHOD(eval)
+ENOKI_CALL_SUPPORT_METHOD(projected_area)
+ENOKI_CALL_SUPPORT_METHOD(max_projected_area)
+ENOKI_CALL_SUPPORT_GETTER(flags, m_flags)
```

Comment:
Let's keep the indentation here

Reply:
This is fixed, I reverted the clang-format pass I had applied.